### ISSUE TITLE
InverseKinematicsStudy() results being written over themselves

### ISSUE THREAD

**Issue by jimmyDunne** [opensim-org/opensim-core#2456]
Currently, InverseKinematicsStudy() does some [automatic editing of the orientations file name-- based on the location of '_'-- to generate an output filename for the orientation results. ](https://github.com/opensim-org/opensim-core/blob/7e4f6ab1e3346f9e822a8a63e008ac16ec25b16d/OpenSim/Simulation/OpenSense/InverseKinematicsStudy.cpp#L190-#L198). 

I have my input orientations files named like this;
```
MT_012005D6_009-quaternions_Walking.sto
MT_012005D6_009-quaternions_RHJCSwinger.sto
MT_012005D6_009-quaternions_LHJCSwinger.sto
```
When I perform an InverseKinematicsStudy() with each of the above input files, the results get written over the top of one another as;
```
IK_MT_012005D6_009-quaternions.mot
```
To solve, we could either (a) don't do any editing of the input string and rather just prepend 'IK_', or (ii) have a property for an output string that the user can set.

**Comment by aseth1** [comment]
I am running into this issue to in my testing of the calibration with different headings. I will fix it as part of m PR since I need it now.

**Comment by jimmyDunne** [comment]
Sounds good. Happy to test :)

**Comment by aymanhab** [comment]
Should be fixed, folded into PR #2459
